components/post-box.tpl

```diff
@@ -22,7 +22,9 @@
     {% if post-box == "article" %}
       <div class="post-excerpt content-formatted" {{ edy_intro_edit_text }}>{% editable article.excerpt %}</div>
       <div class="post-body content-formatted">{% editable article.body %}</div>
-      <div class="post-body content-formatted">{% content name="additional_body" bind="Article" %}</div>
+      {%- assign content_additional_title = "additional_content" | lce -%}
+      {%- assign content_additional_title_tooltip = "content_tooltip_additional_information" | lce -%}
+      <div class="post-body content-formatted">{% content name="additional_body" bind="Article" title=content_additional_title title_tooltip=content_additional_title_tooltip %}</div>
```

Comment:
```suggestion
      <div class="post-body content-formatted">
        {% content 
           name="additional_body"
           bind="Article"
           title=content_additional_title
           title_tooltip=content_additional_title_tooltip   
        %}
      </div>
```

Reply:
Another question as well: shouldn't the `bind` argument be `article` as you're binding it to the article element?

Reply:
In this special context both `bind="Article"` and `bind=article` are correct since it's wrapped inside `{% if post-box == "article" %}` block and the component is called `with "article"` only form [article layout](https://github.com/Voog/design-boilerplate/blob/0d7817d23cab75efa62613e5a220acb151312a8b/layouts/blog_article.tpl#L31).